src/js/components/DittnavLenkePanel.js

```diff
@@ -1,21 +1,120 @@
 import React from 'react';
-
-import Lenkepanel from 'nav-frontend-lenkepanel';
 import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
 import Config from '../Config';
+import { LenkepanelMedIkon } from './LenkepanelMedIkon';
+import OversiktspanelMedListe from './OversiktspanelMedListe';
+import DinesakerSakstema from './DinesakerSakstema';
+
+const MAX_SAKER_SOM_VISES = 2;
+
+class DittnavLenkePanel extends React.Component {
+  makeSaksoversiktPanel() {
+    const { antallSakstema, sakstemaList } = this.props.sakstema;
+    const overskrift = <FormattedMessage id="saksoversikt.overskrift" />;
+    const headerLenkeTekst = <FormattedMessage id="saksoversikt.alle.saker" values={{ count: antallSakstema }} />;
+
+    // const makeFooter = (footerTekst) => (
+    //   <div className="saksoversikt-footer" key="footer">
+    //     <span className="typo-undertekst">
+    //       <FormattedMessage id={footerTekst} />
+    //       <a href={Config.LENKER.saksoversiktHjelp.url} className="saksoversikt-footer__lenke" id="dekorator-bottomborder-overstyring">
+    //         <FormattedMessage id="saksoversikt.hjelp.lenketekst" />
+    //       </a>
+    //     </span>
+    //   </div>
+    // );
+
+    if (!sakstemaList || sakstemaList.length === 0) {
+      return (
+        <OversiktspanelMedListe
+          className="dittnav-lenkepanel-stor"
+          overskrift={overskrift}
+          headerLenkeTekst={headerLenkeTekst}
+          headerLenkeHref={Config.LENKER.saksoversikt.url}
+          border
+        />
+      );
+    }
+
+    return (
+      <OversiktspanelMedListe
+        className="dittnav-lenkepanel-stor"
+        overskrift={overskrift}
```

Comment:
Hvis du prosesserer lista før return slik at det er det du skal sende inn eller null, så kan du kanskje returnere ett objekt der liste={sakstemaList}? Det er bedre for performance under render.

Reply:
Når jeg tenker meg om tror jeg dette kan skrives om/forenkles litt. Denne funksjonen var litt mer omfattende tidligere. Skal se på det.